olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -405,7 +405,7 @@ spec:
                 containers:
                   - name: knative-operator
                     # This reference will be replaced in local builds and CI via hack/lib/catalogsource.bash.
-                    image: registry.ci.openshift.org/openshift/openshift-serverless-nightly:openshift-knative-operator
+                    image: registry.ci.openshift.org/openshift/openshift-serverless-v1.23.0:openshift-knative-operator
```

Comment:
These images are not built yet?

Reply:
It seems that image job failed yesterday. Well, I guess we'll need to trigger CI again first.

Reply:
That should do the trick. https://github.com/openshift-knative/serverless-operator/pull/1587

Seems that config wasn't available for the first run. I could get it now, so :crossed_fingers:. 

https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/branch-ci-openshift-knative-serverless-operator-release-1.23-4.10-images/1529535682949681152